lib/index.js

```diff
@@ -73,8 +72,8 @@ function filterJsDocComments(jsDocComments) {
  * @requires glob
  */
 function convertGlobPaths(globs) {
-  return globs.reduce(function(acc, globString) {
-    var globFiles = glob.sync(globString);
+  return globs.reduce((acc, globString) => {
```

Comment:
You should use `map` instead of `reduce` if you want to generate a new array : 

```
function convertGlobPaths(globs) {
  return globs.map(globString => glob.sync(globString))
}
```

Reply:
That's true, I haven't refactored logic at this stage to be honest, only upgrading deps and syntax.
Speaking of, when I resolve the conflicts I'll try some 

```
const convertGlobPaths = globs => globs.map(glob.sync)
```

Which already questions whether we need the helper :)